_posts/2017/2017-16-01-testing-async-interactions.md

```diff
@@ -0,0 +1,246 @@
+---
+layout: post
+title: "Testing asynchronous interactions in distributed systems"
```

Comment:
What about  "How to test event-based services using contracts". Not the highest score - but it still is pretty much to the point of what the article is about.